package.json

```diff
@@ -33,15 +33,16 @@
     "chai": "^4.2.0",
     "gulp": "^4.0.2",
     "mocha": "^6.1.4",
+    "snyk": "^1.290.1",
     "tmp": "^0.1.0",
     "ts-node": "^8.1.0",
     "tslint": "^5.16.0",
     "tslint-config-security": "^1.16.0",
     "typescript": "^3.4.5"
   },
   "dependencies": {
-    "@adastradev/aws-api-gateway-client": "^0.2.18-beta.3",
-    "snyk": "^1.251.2"
+    "@babel/runtime": "^7.8.4",
```

Comment:
I was getting some strange pipeline errors that were resolved by adding this. I think it is a regular dependency because it's kinda a runtime shim